Sources/Subs-Attachments.php

```diff
@@ -1002,6 +988,52 @@ function parseAttachBBC($attachID = 0)
 }
 
 /**
+ * Gets raw info directly from the attachments table.
+ *
+ * @param array $attachIDs An array of attachments IDs.
+ *
+ * @return array.
+ */
+function getRawAttachInfo($attachIDs)
+{
+	global $smcFunc, $modSettings;
+
+	if (empty($attachIDs))
+		return array();
+
+	$return = array();
+
+	$request = $smcFunc['db_query']('', '
+		SELECT a.id_attach, a.id_msg, a.id_member, a.size, a.mime_type, a.id_folder, a.filename' . (empty($modSettings['attachmentShowImages']) || empty($modSettings['attachmentThumbnails']) ? '' : ',
+				IFNULL(thumb.id_attach, 0) AS id_thumb, thumb.width AS thumb_width, thumb.height AS thumb_height') . '
```

Comment:
nope this one drop the support: https://github.com/SimpleMachines/SMF2.1/pull/3278